### DISCUSSION TITLE
`stream_auth_method 1` Results in Broken Camera Preview Page

### DISCUSSION THREAD

**Discussion by afrazkhan** [Motion-Project/motion#1666] [Q&A]
Hiya,

If I enable auth for the stream URL (by default `https://<ip>:8081/0/stream`), the streaming preview on `https://<ip>:8080` breaks, even though the URL it's pointing to is correct, and clicking on it goes to a working stream (after logging in). Note that TLS is also enabled for both URLs, but I don't think that's relevant.

My guess is that this is because the preview page (`https://<ip>:8080`) doesn't send the authorisation needed for the streaming page?

**Comment by KnKay** [comment]
Same here!

**Reply by mgmaqp** [reply]
in my case i'm using ssl nginx in frontend web and the link for stream is  http://mydnsname:443:8081/0/stream :}(

**Comment by Unturned3** [comment]
Is there a workaround/solution for this problem? I ran into it just now.

**Comment by Mr-Dave** [comment]
Motion uses different ports and possibly different usernames/password for stream vs webcontrol so there is no ready made solution.  

This was changed in Motionplus to fix this but not in Motion.  (Motionplus uses a single port/username/password for everything).  Other alternative is to set up your own NGIX, Apache server.....or just click on the image to enter the userid/password that is associated with the stream instead of the webcontrol.
